applications/GeoMechanicsApplication/custom_geometries/line_interface_geometry.h

```diff
@@ -208,6 +208,26 @@ class LineInterfaceGeometry : public Geometry<Node>
         KRATOS_ERROR << IntegrationSchemeFunctionalityNotImplementedMessage();
     }
 
+    GeometriesArrayType GenerateEdges() const override
+    {
+        const auto points = this->Points();
+
+        // The first edge coincides with the first side of the element
+        auto begin_of_second_side = points.ptr_begin() + (points.size() / 2);
+        const auto nodes_of_first_edge = PointerVector<Node>{points.ptr_begin(), begin_of_second_side};
+
+        // The second edge coincides with the second side of the element. However, the nodes must be
+        // traversed in opposite direction.
```

Comment:
" However, the nodes must be traversed in opposite direction." 
true, but maybe not directly clear.
"However, the node sequence is in reversed order." Is clearer to me, I wonder what others think.

Reply:
I have opted for this wording, since it's not simply a matter of reversing the nodes at the second side. The difficulty is in the fact that we first list the end nodes, and only then any high-order nodes. Simply reversing the list of nodes is incorrect. Consider the case where the end nodes have IDs 4 and 5, and the mid-side node has ID 6. Then from the element's point of view the node order is [4, 5, 6]. Reversing that would give [6, 5, 4], which is incorrect since the mid-side node is now at the front of the list. What I meant to say is if we would traverse this side from the edge's viewpoint, it is in opposite direction compared to what we do for the element itself. Perhaps we can discuss this further in person to see which wording would be clearest? Thanks.

Reply:
Can we then speak about opposite orientation ( with node number according to the opposite orientation )?

Reply:
Thank you for your suggestion. I have updated the documentation accordingly.